pkgs/applications/editors/atom/default.nix

```diff
@@ -24,7 +24,9 @@ let
       sed -i "s/${pname})/.${pname}-wrapped)/" $out/bin/${pname}
       # sed -i "s/'${pname}'/'.${pname}-wrapped'/" $out/bin/${pname}
       wrapProgram $out/bin/${pname} \
-        --prefix "PATH" : "${gvfs}/bin"
+        --prefix "PATH" : "${gvfs}/bin" \
+        --suffix "XDG_DATA_DIRS" : "${gtk3}/share/gsettings-schemas/${gtk3.name}/";
```

Comment:
@jtojnar I guess wrapGAppsHook is more robust here?

Reply:
Yep, now that electron uses gtk3, `wrapGAppsHook` would work. Although, the PATH would need to be added to `gappsWrapperArgs`.

Reply:
alright, will use  `wrapGAppsHook` then
about `gappsWrapperArgs`: what is supposed to be put in there? like `--prefix "PATH" : "${gvfs}/bin"?`

Reply:
it takes the same arguments as `wrapProgram`.